packages/shared/lib/services/sync/notification/webhook.service.ts

```diff
@@ -51,12 +51,15 @@ class WebhookService {
 
         const { webhook_url: webhookUrl, always_send_webhook: alwaysSendWebhook } = webhookInfo;
 
-        if (!alwaysSendWebhook && responseResults.added === 0 && responseResults.updated === 0 && responseResults.deleted === 0) {
+        const noChanges =
+            responseResults.added === 0 && responseResults.updated === 0 && (responseResults.deleted === 0 || responseResults.deleted === undefined);
```

Comment:
maybe something like `const changesCount = (responseResults.added || 0) +  (responseResults.updated || 0)+ (responseResults.deleted || 0)` might be easier to parse than the combination of boolean operators. What do you think?

Reply:
Hmm, I don't think this is that difficult to parse really and i'm just checking if any changes were made not the total count of changes.